en/docs/quick-start-guides/deploy-your-first-integration-as-api.md

```diff
@@ -0,0 +1,198 @@
+# Deploy Your First Integration as API
+
+## Overview
+
+In this guide, you will:
+
+- Create a simple Integration as an API that calls a backend service and returns its response to the client. 
+- The backend service responds with a JSON with a key `message` and the value `Hello World!!!`, and the API deployed in Devant forwards this response to the client.
+- Use WSO2 Micro Integrator (MI) to develop the API.
+- Push the integration to Devant, which automatically builds and deploys the integration to the development environment.
+- Test the integration.
+
+## Prerequisites
+
+1. GitHub account: [Create a GitHub account](https://github.com/signup) if you don't have one already.
+2. If you're signing in to the Devant for the first time, create an organization:
+    1. Go to [https://console.devant.dev/](https://console.devant.dev/) and sign in using your Google, GitHub, or Microsoft account.
```

Comment:
Don't say just  Google, GitHub, or Microsoft. Phrase it as generic. eg:use your preferred signup option